src/query/expression/src/values.rs

```diff
@@ -370,7 +370,12 @@ impl Scalar {
                 let col = builder.build();
                 Scalar::Map(col)
             }
-            DataType::Bitmap => Scalar::Bitmap(vec![]),
+            DataType::Bitmap => {
+                let rb = RoaringTreemap::new();
+                let mut buf = vec![];
+                rb.serialize_into(&mut buf).unwrap();
+                Scalar::Bitmap(vec![])
```

Comment:
This should be `Scalar::Bitmap(buf)`. It's a good habit to run the code locally before submitting it to verify that it's correct!